src/mui/detail/Edit.js

```diff
@@ -20,7 +20,6 @@ export class Edit extends Component {
         super(props);
         this.state = {
             key: 0,
-            record: props.data,
```

Comment:
This was used before we moved the state to `<SimpleForm>`. It is not used anymore.